static/app/components/tabs/tabList.tsx

```diff
@@ -272,10 +274,12 @@ const TabListWrap = styled('ul', {shouldForwardProp: tabsShouldForwardProp})<{
       `};
 `;
 
-const OverflowMenuTrigger = styled(DropdownButton)`
+const TabListOverflowWrap = styled('div')`
   position: absolute;
   right: 0;
   bottom: ${space(0.75)};
+`;
+const OverflowMenuTrigger = styled(DropdownButton)`
```

Comment:
Do we need this styeld component or can we just have the padding in the overflow wrap?